### DISCUSSION TITLE
G helper with proArt laptops

### DISCUSSION THREAD

**Discussion by Cain-cz** [seerge/g-helper#2840] [Q&A] [answered]
So, my question is, does g helper work on Asus proArt laptops? I'm thinking about buying that new ryzen ai 9 HX 370 proArt p16, but I'm not sure if it works on it too.

**Comment by seerge** [comment, marked as answer]
@Cain-cz hello, it does, but usually ProArt laptops support much less things as ROG devices :) (for example they can't disable GPU, so there is no `Eco` mode there)

**Comment by tristan-k** [comment]
So its not possible to disable the dGPU of the newly released ProArt PX13?

**Reply by seerge** [reply]
@tristan-k it is possible, check release notes for latest builds https://github.com/seerge/g-helper/releases
